.circleci/config.yml

```diff
@@ -90,6 +98,9 @@ jobs:
     <<: *defaults
     docker:
       - image: circleci/python:3.5
+    environment:
+      # the tutorial uses Python 3.6 syntax, for convenience.
+      PYTEST_EXTRA_ARGS: -k "not examples/ipython/tutorial_algebra.ipynb"
```

Comment:
This isn't behaving and I can't work out why. These arguments work fine on my local machine.

Reply:
Ah, quoting does not work when expanding environment variables.